spec/services/sort_by_proportion_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe SortByProportion, type: :service do
+  let(:time_entries) do
+    [{ 'name' => 'Hans Meier', 'proportion_used_to_worked' => 0.2 },
+     { 'name' => 'Max Muster', 'proportion_used_to_worked' => 0.1 }]
+  end
+
+  describe 'request redmine for entries' do
+    it 'makes a http request' do
+      expected_solution = [{ 'name' => 'Max Muster', 'proportion_used_to_worked' => 0.1 },
+                           { 'name' => 'Hans Meier', 'proportion_used_to_worked' => 0.2 }]
+      expect(described_class.new(time_entries).sort_by_proportion).to eq(expected_solution)
+    end
+  end
+end
```

Comment:
Two thoughts on that:

* From my point of view you don't have to test `sort_by`. This is a ruby library function an I trust it.
* I think it's not good enough to test it with just two list entries. The test would also be correct if you would invert the list or if you would just switch two elements...